packages/builder/src/builder.js

```diff
@@ -110,9 +110,18 @@ export default class Builder {
           /[^a-zA-Z?\d\s:]/g,
           ''
         )
+
+        // TODO: remove deprecated check in Nuxt 3
+        if (p.ssr === false) {
+          p.mode = 'client'
+          consola.warn(`ssr in plugin options has been deprecated, please use mode instead`)
```

Comment:
This warning could be really annoying as modules should still prefer using it for 2.x compat. I suggest doing warn on N3 upwards.